src/test/common/page/page.js

```diff
@@ -32,7 +32,6 @@ class Page {
   async open (baseURL, lang = 'en') {
     try {
       await this.openDirect(baseURL, lang)
-      this.sessionId = await this.getCurrentSessionId()
```

Comment:
You can also remove the line `this.sessionId = null` from the constructor in this file too.

Reply:
Updated